rust-symcrypt/src/ecc/mod.rs

```diff
@@ -456,25 +484,30 @@ pub(crate) fn to_symcrypt_curve(curve: CurveType) -> symcrypt_sys::PCSYMCRYPT_EC
     match curve {
         CurveType::NistP256 => unsafe { symcrypt_sys::SymCryptEcurveParamsNistP256 }, // SAFETY: FFI calls
         CurveType::NistP384 => unsafe { symcrypt_sys::SymCryptEcurveParamsNistP384 }, // SAFETY: FFI calls
+        CurveType::NistP521 => unsafe { symcrypt_sys::SymCryptEcurveParamsNistP521 }, // SAFETY: FFI calls
         CurveType::Curve25519 => unsafe { symcrypt_sys::SymCryptEcurveParamsCurve25519 }, // SAFETY: FFI calls
     }
 }
 
 // curve_to_num_format() returns the correct number format needed for TLS interop since 25519 spec defines the use of Little Endian.
-pub(crate) fn curve_to_num_format(curve_type: CurveType) -> i32 {
+pub(crate) fn curve_to_num_format(curve_type: CurveType) -> symcrypt_sys::_SYMCRYPT_NUMBER_FORMAT {
```

Comment:
>_SYMCRYPT_NUMBER_FORMAT [](http://example.com/codeflow?start=74&length=23)

Not sure how typedefs work with bindgen, but can you not use the typedef'd `SYMCRYPT_NUMBER_FORMAT` (no underscore)?

Reply:
Changed

Reply:
Don't need to fix this in this PR, but there are other places in the code where the _ prefix is still used. I think the only  reason that exists is because of historical quirks about how you used to have to use `typedef struct` in C. The non-underscore version is preferred, so if you have time it would be nice to clean the other occurrences up in a future PR, that would be nice.